webpack.config.js

```diff
@@ -1,7 +1,7 @@
 /**
  * Webpack configuration file
  **/
-const DashboardPlugin = require('webpack-dashboard/plugin');
+// const DashboardPlugin = require('webpack-dashboard/plugin');
```

Comment:
Yea at one point I was going to add this but ran into some issues. It makes webpack cooler but makes Mac terminal messed up